auth.py

```diff
@@ -15,25 +16,29 @@ def write_passwdb(pwdb):
     with open('passwd.json', 'w') as pwdb_file:
         json.dump(pwdb, pwdb_file)
 
-def pwhash(password):
+def pwhash(password, salt):
```

Comment:
in this implementation, the `pwhash` function needs to be "aware" of the fact that you're using hashing. could you think of a way to avoid this?